railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -21,6 +21,11 @@ source 'https://rubygems.org'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+<%- if options.api? -%>
+# Use ActiveModelSerializers to serialize JSON responses
+gem 'active_model_serializers', '~> 0.10.0.rc1'
```

Comment:
@yuki24 it is the default for now because it is historically much more used in conjunction with Rails API. Jbuilder is mainly used when apps needs an API as a side of the server rendered app and not as an API only thing.

Rails API will work with both. Jbuilder will be just commented out.


@bf4 it's an interesting discussion but I'd said it's better to discuss after merging. Feel free to send an email to the Rails mailing list and/or provide a PR if it's easier for you.